app/views/admin/shared/_image_uploader.html.erb

```diff
@@ -3,19 +3,14 @@
   <p class="mb-6">Quickly upload an image.</p>
 <%# Multipart, remote forms are tricky, the data is actually sent in the Stimulus controller. %>
   <%= form_with(url: image_uploads_path, multipart: true) do |f| %>
-    <div class="form-group">
-      <%= f.label(:image, "Image:") %>
-      <%= f.file_field(
-            "image",
-            class: "form-control",
-            required: true,
-            accept: "image/*",
-            data: { "image-upload-target": "fileField" },
-          ) %>
+
+    <div class="crayons-field">
+      <%= label_tag :image, "Image:", class: "crayons-field__label" %>
+      <%= f.file_field :image, required: true, accept: "image/*", data: { "image-upload-target": "fileField" } %>
     </div>
-    <%= f.submit "Upload", class: "btn btn-primary", data: { action: "image-upload#onFormSubmit" } %>
+    <%= f.submit "Upload", class: "crayons-btn mt-4", data: { action: "image-upload#onFormSubmit" } %>
   <% end %>
 
-  <div class="mt-4 d-none" data-image-upload-target="imageResult">
+  <div class="mt-4 hidden" data-image-upload-target="imageResult">
```

Comment:
When testing the image upload tool I wasn't able to get it to work. Since the class used to hide/show this changed from `d-none` to `hidden` I believe [it should change in this success callback](https://github.com/forem/forem/blob/622dbf004c5244459f0cff079ba81b68209b2ef7/app/javascript/admin/controllers/image_upload_controller.js#L38) too for it work properly 🙂

Reply:
Done. Thanks for this.

Reply:
Also, I have removed bootstrap from `app/javascript/admin/controllers/image_upload_controller.js` file too.